modules/storage.js

```diff
@@ -0,0 +1,6 @@
+export default function storeData(booksArray) {
```

Comment:
> * [ ]  As It's one of the project requirements to use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. 💯
I kindly suggest that you make this into an exportable arrow function.
> * For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)